tests/queries/0_stateless/02564_query_id_header.reference

```diff
@@ -0,0 +1,5 @@
+X-ClickHouse-Query-Id
+X-ClickHouse-Query-Id
+X-ClickHouse-Query-Id
+X-ClickHouse-Query-Id
+X-ClickHouse-Query-Id
```

Comment:
Would be great to have other headers in the test as well, but maybe this is good enough